### ISSUE TITLE
[Feature] hostname check for according to RFC1178

### ISSUE THREAD

**Issue by thorvaldhrafn** [hestiacp/hestiacp#4341]
### Describe the feature or change in detail

Please, add possibility to pass this check or add some new popularity domains like .io or .co to this check. 
This breaks autodeploy tasks when new server hostname is from this first levels domains. 
Panel may be installed manually then with another hostname and change it later, but why it can not be possible to be added in install script?

### Would you like to sponsor this feature to have it implemented?

No

**Comment by jaapmarcus** [comment]
You can provide any hostname:
bash hst-install.sh --hostname dev.hestiacp.com

It checks if the domain has 2 . in it any other valid works fine

**Comment by thorvaldhrafn** [comment]
Yes, if I do it manually, but I do it with Ansible playbook. And it looks like `bash hst-install.sh --hostname {{ansible_hostname}}`. Yes, exists some methods for avoiding this problem, but I want to have not any problem with clear installation.

**Comment by Skamasle** [comment]
Hi

This is an error in your playbook, command not exapand var correctly

Try with:

` bash hst-install.sh --hostname {{ ansible_hostname }}
`

**Comment by jaapmarcus** [comment]
> Yes, if I do it manually, but I do it with Ansible playbook. And it looks like `bash hst-install.sh --hostname {{ansible_hostname}}`. Yes, exists some methods for avoiding this problem, but I want to have not any problem with clear installation.

```
set-hostname {{ ansible_hostname }}
```
Before running installer in Hestia ...

**Comment by thorvaldhrafn** [comment]
Yes, exists many ways to avoid this, but I wrote this feature request to fix that problem, not asks for help how to avoid it.

**Comment by jaapmarcus** [comment]
There is a check if the hostname provided contains at least 2 dots. Doesn't matter if it isn't server.domain.io or any.thing.localhost 

So I don't understand why it should change
